advanced/project-importing.md

```diff
@@ -0,0 +1,86 @@
+# Project Importing
+
+Mimic app can import projects from external file, this feature could be very useful and it is very easy to use.
+
+Mimic doesn't import the file but only save a reference to it, so the path of the file used in this process must not change. For instance this feature if very useful is you save the JSON file with the project settings in the same folder of your project
```

Comment:
The second part of the paragraph: _For instance this feature if very useful is you save the JSON file with the project settings in the same folder of your project_ is not clear to me. Could you please explain?

Reply:
I will explain it better, the idea is that you can share this file with other members of the team with git. So everyone has the same config file for Mimic. I will fix it